### ISSUE TITLE
Anemone crashing SpringBoard 

### ISSUE THREAD

**Issue by elibakshi** [AnemoneTeam/Anemone#201]
I have been having issues with Anemone causing SpringBoard to crash and out into safe mode every so often. I've tried uninstalling everything and then reinstalling it but no good. Please help!

**Comment by coolstar** [comment]
Please see: https://github.com/AnemoneTeam/Anemone/wiki/Debugging-Anemone-Problems

**Comment by elibakshi** [comment]
Okay, thanks! Yes I had everything but Anenome installed and it still crashed. I am using the Lotus theme on ios 10.2. 
AnenomeHTML.dylib is the suspect of the crash

**Comment by coolstar** [comment]
Can you try disabling AnemoneHTML in iCleaner and see if the crashes stop?

**Comment by elibakshi** [comment]
Oh sweet I didn't know I could do that. I'll let you know if it works

**Comment by shadowscott** [comment]
@coolstar I'm getting safe mode upon installation. No themes at all.

http://pastebin.com/QvWbg7Y2

**Comment by RickyX3** [comment]
Disabling AnemoneIcons.dylib stopped the crashing for me

**Comment by shadowscott** [comment]
@rickyx3 doesn't this defeat the purpose of anemone? This disables icon theming

**Comment by RickyX3** [comment]
@shadowscott yes but it stopped anemone from crashing springboard. Its just a temporary fix until a newer update of anemone.

**Comment by shadowscott** [comment]
@rickyx3 that's not true. This has been happening to me ever since iOS 10 jailbreak. Something is incompatible crashing anemone. @coolstar doesn't seem too concerned with helping as I have yet to hear back from him

**Comment by RickyX3** [comment]
@shadowscott have you tried disabling AnemoneIcons.dylib just to be sure? Because your crash report looks just like mine & this started happening since the recent update of anemone.

**Comment by shadowscott** [comment]
@rickyx3 yes, disabling this prevents the crash but that defeats the purpose of anemone. @coolstar should weigh in but I have yet to hear from him

**Comment by RickyX3** [comment]
@shadowscott all we can pretty much do is wait

**Comment by shadowscott** [comment]
@rickyx3 shouldn't have to as it's not crashing on my other phone or other people. Send me your tweak list

**Comment by RickyX3** [comment]
@shadowscott alright, I'll post it on here shortly. do you know what versions of Anemone & firmware both devices have by the way?

**Comment by RickyX3** [comment]
@shadowscott http://pastebin.com/58hDRf1c I'm on an iPhone 6s+ ios 9.3.3
